src/ml-service/packages/ml-api/api/utils.py

```diff
@@ -0,0 +1,105 @@
+from google.cloud import bigquery
+from geopy import distance
+from api import model_config
+
+def get_all_coordinates():
+    client = bigquery.Client()
+
+    query = """
+        SELECT id, latitude, longitude
+        FROM `airqo-250220.StaticThingSpeak.Channel`
```

Comment:
Just a quick one, is this dataset (StaticThingSpeak) still being utilized or is it deleted?